gcs-action/.github/workflows/test.yml

```diff
@@ -0,0 +1,71 @@
+name: 'build-test'
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
+  build:
```

Comment:
Needs unit test invocation, and linter check. See the existing workflow as an example: https://github.com/GoogleCloudPlatform/github-actions/blob/2a9cbc618184da4718e0d16a1b0a055a4f90f66e/.github/workflows/get-secretmanager-secrets.yml#L17

Reply:
Done